lib/sarif/yarn_audit_sarif.rb

```diff
@@ -0,0 +1,76 @@
+require 'salus/bugsnag'
+module Sarif
+  class YarnAuditSarif < BaseSarif
+    include Salus::SalusBugsnag
+    YARN_URI = 'https://classic.yarnpkg.com/en/docs/cli/audit/'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = YARN_URI
+      parse_scan_report!
+    end
+
+    def parse_scan_report!
+      @logs = @scan_report.to_h.fetch(:logs).split("\n\n")
```

Comment:
I think you can simplify the parse_issue code below if you graph the issues as a hash here:

      @logs = JSON.parse(@scan_report.to_h[:info][:stdout] || '[]')

Reply:
We should probably support warn and error too.